docs/src/content/docs/fr/guides/customization.mdx

```diff
@@ -277,6 +277,25 @@ hero:
 ---
 ```
 
+### Désactiver la page 404 par défaut
+
+Si votre projet nécessite une mise en page 404 entièrement personnalisée, vous pouvez créer une route `src/pages/404.astro` et définir l'option de configuration [`disable404Route`](/fr/reference/configuration/#disable404route) pour désactiver la route par défaut de Starlight :
```

Comment:
```suggestion
Si votre projet nécessite une mise en page entièrement personnalisée pour votre page 404, vous pouvez créer une route `src/pages/404.astro` et définir l'option de configuration [`disable404Route`](/fr/reference/configuration/#disable404route) pour désactiver la route par défaut de Starlight :
```

A bit more verbose than the original version but I found "mise en page 404" pretty confusing.

Reply:
Seems more logic!